tests/foreman/maintain/test_clone.py

```diff
@@ -0,0 +1,190 @@
+"""Tests for satellite-maintain clone
```

Comment:
+1, It would be best to move these tests under `tests/foreman/destructive/test_sat_clone.py`, as satellite clone is in itself a separate tool and not related to satellite-maintain

Reply:
@lpramuk Can you chime in here?

Reply:
`satellite-clone` is maintenance tool coming from maintenance repo 
-> this speaks for having it in `tests/foreman/maintain`

but `satellite-clone` is separate from `satellite-maintain` tool
-> this speaks for having it in another location

So this depends how we define tests under `tests/foreman/maintain` - specific single tool   or   generally maintenance dedicated tests

Reply:
@JacobCallahan @Gauravtalreja1 What is your opinion on test placement after Lukas's insight

Reply:
or there is yet `tests/foreman/longrun` :smiling_imp:

Reply:
After the JFT discussion, I am choosing option 2. `tests/foremain/destructive/test_clone.py` for storing my test.